tests/functional/adapter/test_grants.py

```diff
@@ -0,0 +1,52 @@
+import pytest
```

Comment:
Are other grants test cases not included in this PR request?

Reply:
This include all the testcases provided by DBT-core and which are supported.